packages/block-library/src/cover/style.scss

```diff
@@ -167,14 +167,27 @@
 			width: auto;
 		}
 	}
-}
 
-.wp-block-cover__image-background,
-.wp-block-cover__video-background {
-	position: absolute;
-	width: 100%;
-	height: 100%;
-	object-fit: cover;
+	// The extra specificity here helps scope the rules.
+	// The rules have the !important suffix, to protect the behavior from theme CSS bleed.
+	img.wp-block-cover__image-background,
+	video.wp-block-cover__video-background {
```

Comment:
On first thought, I was thinking that it might be better to remove the `.wp-block-cover-image, .wp-block-cover` scope and the `img` and `video` scope, but I haven't thought of a single reason why these styles should need to be overridden since overriding many of them would just break the expected behavior.

`outline`, `border`, and `box` shadow are the only ones that aren't size/positioning related, so maybe the scope could be lightened up on those, but I think the rest need it.

Reply:
Those three rules are all speculative in that I've seen rules that touch those, but not in recent themes. But that's sort of the arms race with the bleed that can happen.

I like keeping the specificity uniform, though — themes really really shouldn't touch those properties.

However I'm also happy that we started out with the other branch that isn't nuclear like this one. I hope that's sufficient so we can close this one! 🤞